### PR TITLE
update doc about customize form item component

### LINKED ISSUE #13609: FunctionComponent support for customized-form-controls
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
https://github.com/ant-design/ant-design/blob/377062bc9b30a0d1cdc7be398577f6b97c7299af/components/form/demo/customized-form-controls.md#en-us

> It must be a class component.

I want to it could be function component with react-hooks.

### What does the proposed API look like?

I want to it could be function component with react-hooks.

### PR DESCRIPTION
**Pull request by zombieJ**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

close #13609

### Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/form/demo/customized-form-controls.md [hunk 1]
```diff
@@ -10,14 +10,18 @@ title:
 自定义或第三方的表单控件，也可以与 Form 组件一起使用。只要该组件遵循以下的约定：
 > * 提供受控属性 `value` 或其它与 [`valuePropName`](http://ant.design/components/form/#getFieldDecorator-参数) 的值同名的属性。
 > * 提供 `onChange` 事件或 [`trigger`](http://ant.design/components/form/#getFieldDecorator-参数) 的值同名的事件。
-> * 不能是函数式组件。
+> * 支持 ref：
+>   * React@16.8.0 之前只有 Class 组件支持。
+>   * React@16.8.0 及之后可以通过 [useImperativeHandle](https://reactjs.org/docs/hooks-reference.html#useimperativehandle) 添加 ref 支持。
```
Comment (afc163) [inline components/form/demo/customized-form-controls.md]: useImperativeHandle 给个 codesandbox 例子吧。

Comment (zombieJ) [reply components/form/demo/customized-form-controls.md]: done. Pls check.

**Review by afc163 — APPROVED** [review]

File: components/form/demo/customized-form-controls.md [hunk 2]
```diff
@@ -10,14 +10,18 @@ title:
 自定义或第三方的表单控件，也可以与 Form 组件一起使用。只要该组件遵循以下的约定：
 > * 提供受控属性 `value` 或其它与 [`valuePropName`](http://ant.design/components/form/#getFieldDecorator-参数) 的值同名的属性。
 > * 提供 `onChange` 事件或 [`trigger`](http://ant.design/components/form/#getFieldDecorator-参数) 的值同名的事件。
-> * 不能是函数式组件。
+> * 支持 ref：
+>   * React@16.8.0 之前只有 Class 组件支持。
+>   * React@16.8.0 及之后可以通过 [useImperativeHandle](https://reactjs.org/docs/hooks-reference.html#useimperativehandle) 添加 ref 支持。（[示例](https://codesandbox.io/s/31mv8004rp)）
```
Comment (yesmeck) [inline components/form/demo/customized-form-controls.md]: 用 `forwarRef` 就可以了，不需要 `useImperativeHandle`。

Comment (afc163) [reply components/form/demo/customized-form-controls.md]: https://codesandbox.io/s/m75y819qlj

**Pull request merged**